content/cloud/data-redis.md

```diff
@@ -120,31 +120,32 @@ The following is the content which is required in the `src/lib.rs` file. Feel fr
 <!-- @nocpy -->
 
 ```rust
-use anyhow::{anyhow, Result};
-use spin_sdk::{
-    http::{Request, Response},
-    http_component, redis,
-};
+use anyhow::{anyhow};
+use spin_sdk::http::{IntoResponse, Request};
+use spin_sdk::http_component;
+use spin_sdk::redis;
 
 const REDIS_ADDRESS_ENV: &str = "REDIS_ADDRESS";
 
 #[http_component]
-fn publish(_req: Request) -> Result<Response> {
+fn publish(_req: Request) -> anyhow::Result<impl IntoResponse> {
+
     let address = std::env::var(REDIS_ADDRESS_ENV)?;
+    let conn = redis::Connection::open(&address)?;
 
     // Set the Redis key "spin-example" to value "Eureka Cloud!"
-    redis::set(&address, "spin-example", &b"Eureka Cloud!"[..])
+    conn.set("spin-example", &b"Eureka Cloud!"[..].to_vec())
```

Comment:
I vaguely recall we found `into()` worked?  If so, it might be more concise than all the `b...[..].to_vec()` malarkey

Reply:
I found that into() generated an error, so I switched back to the `&b"Eureka Cloud!"[..].to_vec()` which worked.